src/index.ts

```diff
@@ -207,6 +160,43 @@ function Loadmill(options: Loadmill.LoadmillOptions) {
         );
     }
 
+    async function _getExecutableTestSuites(): Promise<Array<Loadmill.TestSuiteDef>> {
+        const { body: { testSuites } } = await superagent.get(`${testSuitesAPI}?rowsPerPage=100&filter=CI%20enabled`)
+            .auth(token, '');
+        return testSuites.map(ts => ({
+            id: ts.id,
+            description: ts.description
+        } as Loadmill.TestSuiteDef));
+    }
+
+    async function _runAllExecutableTestSuites(
+        options?: Loadmill.TestSuiteOptions,
+        params?: Loadmill.Params,
+        testArgs?: Loadmill.Args): Promise<Array<Loadmill.TestResult>> {
+
+        const suites: Array<Loadmill.TestSuiteDef> = await _getExecutableTestSuites();
+        const logger = getLogger(testArgs);
+
+        if (!suites || suites.length === 0) {
+            logger.log(`No test suites marked for execution were found. Are you sure flows are marked with CI toggle? - exiting...`);
+        } else {
+            logger.verbose(`Found ${suites.length} test suites marked for execution. Executing one by one.`);
+        }
+
+        const results: Array<Loadmill.TestResult> = [];
+        for (let suite of suites) {
+            logger.verbose(`Executing suite ${suite.description} with id ${suite.id}`);
+            suite.options = options;
+            await _runTestSuite(suite, params)
```

Comment:
Why not run all suites in parallel?

Reply:
It will fail if your domain isn't verified and if your tests can't run together (i.e. some tests expects no data in the env etc...) 
We are thinking of allowing this in the future but right now customers don't need it